JSDemos/Demos/TreeView/LoadDataOnDemand/description.md

```diff
@@ -1 +1,9 @@
-In&nbsp;those special cases when the built-in Virtual Mode fails, you can gain full control over data loading in&nbsp;the **createChildren** function.This function is&nbsp;called each time a&nbsp;user expands the TreeView node that has not been expanded before. In&nbsp;the **createChildren** function, you get data from a&nbsp;server or&nbsp;other sources and then form the collection of&nbsp;child nodes from this data.
\ No newline at end of file
+Use the [createChildren](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#createChildren) function to load data on demand and enhance performance on large datasets. Unlike virtual mode, this function allows you to use your own custom logic. 
+
+To specify the function, pass the expanded node's identifier as an argument and process the returned value inside the function. This value can be an array of objects or a promise as in this demo. The following prerequisites should be met:
+
+* Neither the [dataSource](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#dataSource), nor the items property should be specified.
+
+* The dataset should have a plain structure and the [dataStructure](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#dataStructure) property should be set to *"plain"*.
+
+* The TreeView should not add the [search panel](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#searchEnabled) because the search is performed on loaded nodes and may produce incorrect results.
```

Comment:
It is a little bit weird to see that TreeView should or should not do something.  Maybe rephrase some of the list items in imperative mood?

- Do not specify the items or dataSource properties.
- The dataset should have a plain structure and the [dataStructure](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#dataStructure) property should be set to *"plain"*.
- We recommend that you do not [enable search](/Documentation/ApiReference/UI_Components/dxTreeView/Configuration/#searchEnabled) because the search is performed on loaded nodes and may produce incorrect results.